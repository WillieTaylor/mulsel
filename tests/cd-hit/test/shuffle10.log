================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.90.out -c
         0.9

Started: Fri Dec 11 18:04:27 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        364  clusters

    10021  finished        365  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.80.out -c
         0.8

Started: Fri Dec 11 18:04:27 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        187  clusters

    10021  finished        187  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.27
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.70.out -c
         0.7

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        120  clusters

    10021  finished        120  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.28
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.60.out -c
         0.6 -n 4

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 3M = 3M
Miscellaneous   : 0M
Total           : 15M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98011432

comparing sequences from          0  to      10021
....................    10000  finished         84  clusters

    10021  finished         84  clusters

Apprixmated maximum memory consumption: 15M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.50.out -c
         0.5 -n 3

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 0M = 0M
Miscellaneous   : 0M
Total           : 12M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98381872

comparing sequences from          0  to      10021
....................    10000  finished         63  clusters

    10021  finished         63  clusters

Apprixmated maximum memory consumption: 12M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.40.out -c
         0.4 -n 2

Started: Fri Dec 11 18:04:29 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 0M = 0M
Miscellaneous   : 0M
Total           : 12M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98399512

comparing sequences from          0  to      10021
....................    10000  finished         37  clusters

    10021  finished         37  clusters

Apprixmated maximum memory consumption: 12M
writing new database
writing clustering information
program completed !

Total CPU time 0.24
