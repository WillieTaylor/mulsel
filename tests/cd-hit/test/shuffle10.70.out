>seqP74 = P family seq 74  0.005087 69 P
PPPPPPWPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP
>seqN47 = N family seq 47  0.005802 72 N
YNNNNNNNNNNNNNNNNNNNNNNNINNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>seqI519 = I family seq 519  0.005806 34 I
INIQIVIIIVIEMTYIIIIPFPMVFKIIEKVIIIQTVIVEENTIIKPNIIILIIINCVVDILIRMICIIISIIGIRDVE
>seqK376 = K family seq 376  0.006141 34 K
MCKKGKRIYKVKTKPKAKKKTKEKKKDTYSKKKIKKKKGHMCKKHHKKPSKKDKKTRIKVNKKCKKDEAECM
>seqI279 = I family seq 279  0.007653 40 I
NCVVIRMMIIEIIINIRILISGAVWIIIIVMIIEPIIIIIIWIIIIIMIIIINQIICILIMINIIRIIIGFR
>seqT178 = T family seq 178  0.009715 66 T
PAICTTTTTLTTTTTTTTTTTITTTYTTWTTTVTTTTRTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTNTTTTTYTTTV
>seqM120 = M family seq 120  0.010638 71 M
MKMMMMMMMMMMMMMMMMMVMMMMWMMMMMMMMMMMMMLMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
>seqC58 = C family seq 58  0.011272 70 C
CCCCCCCCCCCCCCCCCCCCCCCCCCCCRCCCCCCCCCCCCCCDCCCCTCCCCCCCCCCCCCCCCCCCCCCCC
>seqK41 = K family seq 41  0.013765 77 K
FKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK
>seqM261 = M family seq 261  0.015076 50 M
MKPMMRPMMMMMMSCMMMEMMVMMMWWMMTSMMMMMMMMMTWQMMMCMWSPMMIMMMMMMMVMMMRMMMMMARMHQPDD
>seqY598 = Y family seq 598  0.017382 34 Y
YMERLELYYYYYYYLQAYYYYKSYYYGEQIYDYYLETAYYEYDTHHIYEETYQHHYGQGYYTYEAYYGEMRYEYYYYN
>seqE596 = E family seq 596  0.018960 36 E
DKEEEHHYDCWENGEGTEVEEAEEESEENNEREEEWEPQKDEECEETEWEHEVAETTKVYEENEEIEIEIQEELEEAKYW
>seqE265 = E family seq 265  0.018974 44 E
KCGHLEEDEEEEFEGECEENVDMEEEWEEEQEEEEEKIPEEFEEEQLTVEEEEEEEECEWETLVENEWEEEFEYEELQFK
>seqA28 = A family seq 28  0.019433 74 A
RAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
>seqE496 = E family seq 496  0.021080 34 E
FEEEHHEEEEEPEEEGTIVIIHAQESTFEEVEYEEEEEPEKIECCTKEVEHIVPEQTEVDEENDQIEEIELEEAKE
>seqY213 = Y family seq 213  0.021769 54 Y
APNCYALYYYYYYYYTYYYYPYYPYYLYYYFYYYWYYYYYYYYYVYTYYYYYYYYYYYYYYVWYYYYYIYLPYALWKW
>seqM434 = M family seq 434  0.026963 34 M
YYTWCQDYGMRMIMWMMNIMREMMMMMWWAIISFMMMMMMMAWPVMMNMMSLARWDMRFMMMMMMMMMMVARMYHQTIN
>seqV466 = V family seq 466  0.029478 36 V
GVVRVWVNVLCVENVEHSNVVVTVVVLGAVRVVRQFVHPDVTVVVVVVWPKDVVFVNVVVVVSVESRNGAVVV
>seqL587 = L family seq 587  0.030079 34 L
LQIIALDLLLLHELLTRCLMCWQVLLLELLCERLHQLTAQWLLKLLYLLLLQHLLLGFLLLYPLQLYHLLTW
>seqV24 = V family seq 24  0.032825 75 V
VVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVV
>seqT305 = T family seq 305  0.035831 38 T
AKILTTTTTTCDTTGKTITTTATLWTTTGPNTRTWHTTNTTTCLTWTTTKTTTTTTTEIATTNTKFNAYTHHGVA
>seqS169 = S family seq 169  0.038930 64 S
FNSESSSSSSSSSSSSSSSLSSSSSSSSSSSSSWASSKSSSSSSSSSSQSSSSLSSSSNSSSSSSSTESSSSSSSSVN
>seqA371 = A family seq 371  0.039906 34 A
AKYANKPAHAAKAATALAASAQAAAVGAAAAASVCYAYAAAASIAASGIAAAAAECAVAASSMTYDIFNT
>seqK542 = K family seq 542  0.044763 35 K
MIKKKKKGKKKKPHKEVVKKKKKKEKYQTVYSKKKNMKTNHMCSAHKKSSDEKTDTIDKVNKDKCKKKKNKKKK
>seqM340 = M family seq 340  0.045554 34 M
WIDPGMRMMIMHMMMCMMMEMMVMMIWWAMTSGMMMMMMIWQPVDMNMWSPAIMNFMMSMVMMMMMNMVARMHYHQPND
>seqH592 = H family seq 592  0.054113 34 H
HPRHDWHHCPLHHRHHHHQHHCHHCHGVTHHHHHDIEPHTHCHNECDHHHNYDFAHIYHHPGYMHHHNAFHYLTHDT
>seqP472 = P family seq 472  0.054910 35 P
NLNPMFLKNPPPQPPQPPFPQFNLNWLGPSQPWGLLPYNQHTPPVHPFLPPLPQPPPTEPPPPPWPPPKPSKPPPPP
>seqR370 = R family seq 370  0.056344 35 R
IYFTDMPLRRPSTRMRARRRRDYIERKRVLRKDRRTTPGRRRIRVTIDRKRRRMQRRRRMVNRRYERRRMRRKVGLRPRQR
>seqN468 = N family seq 468  0.057369 34 N
NNNNRNNNCNDNNPFNNPNNNIETNANQRWHQCNEINIVNNSQNINGFCENEMNNNRNNFNNNSMFISYNIQSRNM
>seqL365 = L family seq 365  0.058535 36 L
MTLLEFWLLAELLFEELCLLVNWLMCGYLLLLQLNMLEALSLLIQDLPLLLSKLLLMHLHLCLLQLHYGLLYLLMQLQ
>seqS304 = S family seq 304  0.060512 39 S
FRNVSELSIGSSSRSKSWSSLSRSSSWQSRDSSKSMILKSSSSLSSYQYSSSLSSINNSSSSAGSSKSYASSSSYAVN
>seqF390 = F family seq 390  0.062097 34 F
QFPKWFFFFFFVFDFKYFYFFFFFMFLTFQLAFIQLRSNFGFMLDFFDWAWFYFTFRSFFFLFFVFKYYTIMQFFLFPQRW
>seqE427 = E family seq 427  0.063793 36 E
SIEEHHESEPEEEEEGTIEIIHAQESWEEEVEEEEENIPEEFEEIATVQDEEEHIEPEYTEESENDCEEELEEANK
>seqI169 = I family seq 169  0.066892 64 I
CTIIIIIIIIIIIIIILIIIIIIIIIEIIIIIPIIIIIIIIWIIIIIIIIIIIIIIIIILIIIMIIIIIIIIGAF
>seqC359 = C family seq 359  0.067600 36 C
PGCCCACCARKCNLCACCRCFMRCCHCCACCCNMSCDCCCCCPCDCNCSCCCSCWWCNCNCAANNCLQCCTYLR
>seqM594 = M family seq 594  0.071724 34 M
QMLMMCQHLDYMMMMWHMMKPRMEPELMWPMISFMMMMKMRMMMNMLLAKVHVMMMMMAWCHMMMMQMMMMPGSN
>seqA571 = A family seq 571  0.074108 36 A
FARYYAYKAYAALNNAAAAAGACLAAAAARAAFQAAEYAAIICKVAIDRAVAYACKLAQAAYSEAQNSALGRSLANLCAAAGA
>seqE145 = E family seq 145  0.075331 64 E
KIEEEEEEFEGEEEEEEWEEEEEEEEEEWEEEEEKEEEEEEEEEEEVEEEEEEEEKEEEEEEEEWEEEEFEEEEEQK
>seqK302 = K family seq 302  0.081489 35 K
LIHPKKKRIYKKKPKKKKSNDKQKKKETYSKKKIKKKKKMKKKHKKSKKDKKYRIKVNKKIWKDCAER
>seqS474 = S family seq 474  0.087538 34 S
STRSNSVVESLSSGFNSEVGSSMSSWNSRSTQSSMSSSSSSILIGSKSVPLSSYQYGDMGGGSSLSSSVSCASSYASVN
>seqS595 = S family seq 595  0.089147 34 S
SSSRFSYSSSENSFGSNNAFSNSTYMIVSSITSKSMSGSFSSSSSSTVSIYAKDSGSSLISSLSSTHSISNSN
>seqF540 = F family seq 540  0.095715 34 F
DFFFFWVFIFLFFDVFFFSFFNFFRFMQFPFFFRLFFFFCDFTWFIWYFQPFFNFLFEFKEETMQPFLLFPQRWQ
>seqI594 = I family seq 594  0.095723 34 I
WIIIWKGIKIVICEMYIIIIIIFRIVILIEVIIMIIVNIEENTNIKPLIIIFILNAVVDILGIRFICIIIIIKIRDVE
>seqW540 = W family seq 540  0.099026 35 W
WWKWWWPQWWWWNWWFFWWSTQFLWWWLWPNNACPWWFRWKVMFWWWWPPQCWWFVWWNWDFENCWWGEWWGWFIWPM
>seqL198 = L family seq 198  0.106961 60 L
MTMTWLLLLLLLLLLLLLLLLLLLLCLGLLLLLLLLLLLELLLLLIWLLLPLLVLLKLLLDLLLLCMLKLLLLLYLLLKW
>seqV372 = V family seq 372  0.107828 34 V
GMVKVWWNVLVCVVVVSTVVVTTVVVLWAVDVVNVTVHVHTVVGYVLWCKDVVVFQNVVVFRVVVVSRNGAVW
>seqD364 = D family seq 364  0.112264 34 D
SDMDSDDDDDCDDDIANDIDPWDWDEDSDRRRDCSDSYDDRIHDKDDQWADRHWHIMDSIDDDIRDDDDKDD
>seqW163 = W family seq 163  0.115366 64 W
WWWWWNWWWEWWAWWIWWWVWWWWWWWHWWWWAWWWWWWWWWWWLWWWWWWWSWWWWWWWWWWWPWAWWWWWWWWH
>seqR597 = R family seq 597  0.117120 35 R
VCRRFDFSRRMCYRRRRNTFVQVIRRHRARKDRMFAWGARYRWRRRRRRRAKVRRQRFATRHRVEFTRRRFRRTVRRTFPRQK
>seqT461 = T family seq 461  0.117601 34 T
TMTCTFLTTRTTLDATGTHTTPTLWTTDTYTRTCHTRNKYTTLTSTTTTKTPTTELTTNMKMRTYTYTCTVMMFIG
>seqQ151 = Q family seq 151  0.121187 64 Q
RQQSGQQVQQQQQQQHSQQQQQQQQYQQQQQQQQQQQQQQQQQQQQQQQQQDSQEQQQQQQQQQQQQQQQWQQQQPT
>seqN570 = N family seq 570  0.127200 34 N
NHNNHRNDNCNWQNPNNNNPNNIVETHANQHWHMNLNNVANNNSNINGHENELNNNRLNVNDWNTNANFAEYNTWSNFN
>seqF475 = F family seq 475  0.129885 34 F
DFFFKWFIFFVFDFFFYFFFFFRFLQLAFFILRLFFFFFDDFRWFIWFFTPFFFLFKSTMQPFLFPQRWF
>seqD451 = D family seq 451  0.133209 34 D
DDLDSDASDFCDLDMANDDDDPWDQDEDVDRRADCLDFSDDDRIHDDKDYQASADDDWQIMDIDMDIRQDSDRDDD
>seqQ412 = Q family seq 412  0.134461 34 Q
QGPHVQQQQVQSQELHHVFMQQQQDQFQRYQQQMQQNAWQGQQQKMWDSQLQQADGQHQQQQAQQQEQDRTGPG
>seqW420 = W family seq 420  0.135434 34 W
WPPWWNQWCWWNWWGVLVGWMIWWCWWKWLHWKNAWWRWWRWKWWWEPPQCIWWWKDWNWDVFWNCWWWWVGWFIG
>seqG318 = G family seq 318  0.138492 34 G
FYNGGPGGGWGGGGGAGGGHHGEWGNSQQGYGGASPGGGGGGSKGGCAMGGGLGWGGRLAAGPELKQIPFGQPQF
>seqP241 = P family seq 241  0.139677 46 P
DDYPPPPLPWPPPPPPPDFPPPPPRPKPPPPPPPPFPYPPPQPTMYFPPPLPKQFPPPTEPPPPYIPPPPQL
>seqW299 = W family seq 299  0.152807 35 W
IPPLWWCHEWWWQLKGWWIWWWWWKWHWWAFAWWWHWCWTWWWIWPWRCWWAWWMHWAPWACYWWWWHVIR
>seqG377 = G family seq 377  0.154756 35 G
FYGNGGGGGGGNWGGHTGAPGGHGGEWGWSQQGCPGGRSPGGGGGTKGGCMNNGGLRWGGRGGLAGPELKQGPFQPFQG
>seqL488 = L family seq 488  0.158303 37 L
ELTLEFWLLLLDELLLLELLDVDMCGYLVLLLYLERLSLLLAQCDLLLLSGLLSHLLLCIYLLYGKYLLLLLP
>seqT558 = T family seq 558  0.162091 36 T
MTQEFTTTLCTRTSDTTETRGTTTTDPTCWTTTQYRTTCHCTNRCTTLPFTETPFTTETLTMTCTETRQTTSCTMHCDTT
>seqA256 = A family seq 256  0.163760 41 A
SYKYANKAVHAAAAATAYLAAHAAKAAVATAAAFAAAVAYAYAAIAAAAADNAAAAAAECAVCAAAAQYYFNN
>seqL547 = L family seq 547  0.168878 34 L
LIIFIDLKLELLELTRLDMCLVLLELLLEILTGLTAQCLLLLLDLLLLSLLLCGFLLLLYPLLLTIPF
>seqI387 = I family seq 387  0.172230 34 I
NICGVCAIKMITYEIIIIIRFIIIKIIIVIIITPISQEIPYKIIQTMIISINQIVCILIMIIRISIIGIIFYE
>seqA499 = A family seq 499  0.177212 34 A
FARYHAKYAALANALAASGAALAAASAARAAFAAAYAKEVCMAARYKAYACKLHQAYAALQLAAAGRSSYLTARIGAA
>seqY471 = Y family seq 471  0.193279 34 Y
YMYYGRYFKYDYRTYYSYYYYVNISDYYYYTYYYYDVYIYYYYHHHYYYYYCRGYYGYLMRGHYELN
>seqD514 = D family seq 514  0.204814 34 D
DLDIASDDDFDIMALRDDDQMQESVKDRDIDCLDQSDDIDRIDDDDQYDADADQWEIMDDDDMDIDSQDDIDDD
>seqE318 = E family seq 318  0.212896 34 E
KICIHREEDFEEEYEGTEWIDLEEESWEEEVEEEWEKIPEWFEIATVQEEEEIECEYTEVESEWETEFEYEELQEAKI
>seqN229 = N family seq 229  0.213590 49 N
YNENNQLNNNNNNNNNNNENNNNIRRNGELNNNPNNNGNQNNNNNNNYNCNNVENNNDNNNNSNVNNNQNQNP
>seqR454 = R family seq 454  0.216649 36 R
QVRFTDMRRRRPIRRMRAECRRDIERRARKDRRRAPGARRIRVTRHDHRRKRRRQRARVRMVMYNRQARERDLRGRRPRQR
>seqD573 = D family seq 573  0.222011 36 D
MDDDDMSASDDTDDIEDLRCDDIQMQESVDDEDTTCLDTSQDDDDIDDDQYDAPADQDEDMDHDMDKIDDQDDIDDD
>seqQ588 = Q family seq 588  0.225145 35 Q
LQGVRRQVPQQQELNLQQQQQGDRQQQVQQQQSDQGAQQWQIQQEQVMQDVYQYAGQQTQQQCQQRQDRTVQPGY
>seqV207 = V family seq 207  0.227293 48 V
WWNVVVHVVDVVVVVVVVVPDVTWVVVVVVAVVDVVVNVVVYVHVHTVVYVVWCVDVWVVWNVVVHVVWVVVIVN
>seqT391 = T family seq 391  0.236609 34 T
TKCITFLTTTTTLDTTTGTTITPPMTLWTTTGYNTRTIHTTNKYTTLTSWTTTKTITTTEILTTNMKTNAYHPHGVMFIG
>seqG197 = G family seq 197  0.242544 52 G
YNSGGGRWGGGGGGGGHGSGGGGLLKGGGGGGGGGGGGGGGGSGGGGMGPGGGGWGGGQGLGGGPGKGGIPFGYL
>seqK490 = K family seq 490  0.243408 34 K
MIKKKKGKKKKVKKPKVKKKKKIEKKYQTLSSKKNMKKHMCKKANHLKPSKDEKTDIKKVNKKCKKDKNECM
>seqG451 = G family seq 451  0.244953 35 G
GGNGGGGGGGQHHGAGGGGGEWSQHQGGPGGRSPGGGGGKGTNGGGWGLRGPGQHGPEHTGGPFQKFG
>seqF167 = F family seq 167  0.247961 64 F
RDFFFFFFFFFFFFLFFFFFWFFFFFFFFFFLFNFFFFFMFFFFSFFFKFFFFFFFFFFFFFFFFDFFFFFFFFYC
>seqR533 = R family seq 533  0.248142 34 R
QCRSFDRSRRNYRRMFNEFRRVIRRRARKDRFARGARYTRPTRRRCRYKRRRQRFARRMVMYRRFRRTVRRDRPRQK
>seqF302 = F family seq 302  0.251599 44 F
KKWFFFFFYFFFFFKYFFFFFFMFKFGFAFIDLRFNFGTGFDFFDWAFKYFFFFFFFFFFVFFYYFIFFFFNFQKW
>seqR258 = R family seq 258  0.272780 53 R
IYFIRPPRRRPRRRRRRRRRDRIRKRRRSRRRRFRRRRRRRRMRRRRRRRRSQRRRRRRRRCRYVRYRRRRRKVGVQ
>seqK445 = K family seq 445  0.276123 34 K
MIKKKRKGRKKKVIKKPKMKKKIKVKIEKKDTLSSKKIKMKLKGHMCKKKNHRKPSIDEKTKIKNVNKKKKKDVNECM
>seqM486 = M family seq 486  0.312235 34 M
QMYMCMQQDYEMMWMMWMMNKIGRSMPHMMWWAIISFMMMMMMHWSPVMMNMMSLARKDHWMMMMSMMMCYMMMQRMWEHQTNM
>seqH319 = H family seq 319  0.313816 37 H
PMMEWVPHHHHMHHHHMHHMIHRHDHHHWHDHHPHHHRWKKHHHHHFHHHHNTHHHGRDHKHNHAHRMRNKK
>seqP367 = P family seq 367  0.319171 35 P
RYMLKPPPPQLPTPPPPPQFNPLPRPKIWPPQPPHGPLPPQHTPYPHPFLQFGPPTEIPPPPFIPPSPPPSMHP
>seqY348 = Y family seq 348  0.320435 34 Y
AYRCYKYDYYTYYYPYYSFLNVIFYYYYYYYYYYYGITYEYYYYHYYYSYMKWCYYYIYLPFAEWGWN
>seqV534 = V family seq 534  0.331809 35 V
GYVEVWVVVTLECCVEVVVHCNVVVTIVVLGYRPVVQFVVVNVTVVQVVRPFVDVVQTNVVVVSSRNGAVVVV
>seqL440 = L family seq 440  0.335667 34 L
ETLLEFWLLGAELLLELCLLVMCGYLVDCLLYLERLSALLLQLGLLLSGLLLLHLHLCLYLLYGKYLLLKP
>seqL299 = L family seq 299  0.348112 36 L
MTMWWLLLLLLHWLLCLLVLWMCGLTLLLLLLMLELSLLICQALPLSSKLLLMHLHLCMLKLHLLLYLLMQ
>seqS525 = S family seq 525  0.356883 35 S
SSRFSNSSESLSSGFNSFGNSTSSSVSSITQKSMSSSSFTTSSKSVLSSSQYGKDMGGGSSLSRQSSSTSYISSN
>seqP533 = P family seq 533  0.371432 34 P
NNPGFPGNPPPTPPPFPQFPFINCLNWLPHQPWGLPPYNQCTPPVPPFLPPPPQPFPTPPPPPKWPPHKCSPMPMP
>seqH237 = H family seq 237  0.371672 54 H
MCVHHHHHMHHHHHHHHHHHHHHHHHHHVHHHHPHHHHRHKHHHHHHFHHHHHNTHHHHGFHHHKHHPHRMRK
>seqV599 = V family seq 599  0.379219 35 V
NYVMVSQTHQCEVVVHCNVVVTVHVVLGVSVPVLVVVVVVCHQTVRCFVDVVNVVVPVVSVLSSVHSGVVWVV
>seqN344 = N family seq 344  0.396160 34 N
NEANNNQLNNRNNNNNNNNENCNQLNGELNNPENNGSQNNQCEWMNNVENNNNSINNNQNQNPSPLCM
>seqT505 = T family seq 505  0.396160 34 T
MTMEFTFTLTTRTSDTAESGTTTTDPTCWTTDTYTTMCHTTNSCTTLMFTTTTKTPHTTHELTMTNTKHRTYTYCTMHFKIG
>seqH475 = H family seq 475  0.416910 35 H
HPHRHWNHWHLHKHHHLQHWTHCAHHIHHHHHWHDIWPHTHHACHHHFHIAHAAMGHHHNGRAMHHHNAHLTIRTKAHK
>seqQ504 = Q family seq 504  0.422949 34 Q
LQGQRQQKVVQQQELQQFMFFQGDRQRYQQCQQDQNKQWQGQKEQKMWDVQQQFQADGQQTQQQCQQRQQDRTLQPGK
>seqD256 = D family seq 256  0.434837 43 D
NIMEWDDDDDRCRDDDIDDDIDTWDWDDSDDRDNDCPDDDDYKDTDDDKDDDWDDRDWDIMDYDDDDIDDDDKY
>seqK199 = K family seq 199  0.447612 49 K
IAKKKRILKKKIKKKKSNDKKKKRKKKEKKSKKKKIKKKKKKKGKKHKKKKKKDKKKKKKKVEKKKIYKE
>seqC449 = C family seq 449  0.464546 34 C
CKYAGCRGACKMNLCCCRCFMRCCQCCACCLNMSCCCCCCCCDHCSCCCMCCRKYYNCAAYCCLQCTCCYLC
>seqG596 = G family seq 596  0.488609 34 G
GEGYGGGGGGTGLGYWPEGGSHPGGKPGSSHGGGGGCGKNKPTGSLFLGGGHGPGHGGGGGYGEFG
>seqY412 = Y family seq 412  0.512473 35 Y
ARYRRYFKYDYTYYYYCYYSYNISFYYYYTYYYYYIYEYYYYHYYYYYCWYYYYLPFQEWEYN
>seqG522 = G family seq 522  0.512779 34 G
GGGGGGGQTHTGALGGGDKAEWSSHLPGGPGYSHGGGGKGGSNGDGPGLRGPGGHGGPNHGGGPFGYGFG
>seqD131 = D family seq 131  0.544802 67 D
NIWDDDDDDDDDDDDDDDDDDIDDDWDDDDDDDDDDDDCDDDDDTDDDDDDCDDDDDDDDDDDDDDDDDDDDDDDY
>seqW360 = W family seq 360  0.559319 35 W
WPPWLPWCHEEWWWQLVGWWITWWGWWKLHWRNAFWWRWWRWKWWWIPPWRCIWWKWWMWDVPWWCPWWWWHVGWWIR
>seqQ259 = Q family seq 259  0.609861 45 Q
QQVLQQVTWQQQVHSQMQQQVDAQCYQQQQQQQQQMQAQARQWRQQQQQKQMQDSQLEQQQYFQQQSQQQQQEQHTQPL
>seqN403 = N family seq 403  0.633863 34 N
NINNVNNNCNLNNRFNNNPNNNNENCNQRWHQCNENPIVNNNNSQNIWQCELEMNNVRNNNNNSMMISQNINPHSCRHM
>seqH534 = H family seq 534  0.634461 35 H
PRVHWHWHLHHRHLHHWQHRCHHHHIHVHHHHHDIPHTHCHHCHHHRYFHIAHIYGHHMPGMHHNAHIHVKTHGM
>seqQ353 = Q family seq 353  0.642107 34 Q
QHVGQQQVTSQELHHVYMQQQVDAFCRYQQSQLQQQNQASARQQQKQKMQDSQLEQMQADFQHQQSQQQQQEQHRTQQPL
>seqF593 = F family seq 593  0.647475 35 F
DFQHFFFVILFFTVFFNVFFNQFFMTFQFPFFFLNFFFHCFFIFVIWYFQFFMFLFQFKFTEVFMQFLLFPCQFWQF
>seqA426 = A family seq 426  0.665350 34 A
AQSAKYANKAAAAAAAAVAFLAACSAQAAAVGAAAAASVCMAANYMAACIKQGIAAAMAECAVAGRSSMSTAVIGMT
>seqH434 = H family seq 434  0.691156 35 H
HPHHWHPHKHHHLHHHKHKATITHTHWHDIHPHHAHKHHHHHFHKAHAAMGHHHHGRDMHHKKNAHLTMRTKHH
>seqP593 = P family seq 593  0.700561 34 P
NRPRPPCPGNPPPTPPPLFPFPIPCLPWPMHQWGCLPNQCTPPVPPFPRPPPPQPFMTPPPKRPHPPPMP
>seqS394 = S family seq 394  0.739591 34 S
FRSNSVVELSIGSNSSSSSSMMSSLSFRDSWQSRMSSKFILKGSKSSSLSSYQYSGSLDMIGNAGSSLGSSSVSCASSCYASVN
>seqN507 = N family seq 507  0.745134 35 N
NHNNHRNNNCNDQNPNNNYPNNIETNANQNWHQHNENIVANNNNSQNIGFQENEMNNRHNVFNDNNSNNFASYNILSNMN
>seqI468 = I family seq 468  0.928023 34 I
INIPMIVIIEMTYIIIIIPFPMVIIKIIKVIIIQTPISEEIIIIKPIITMIISNQVVDILIMIPRISIIGIIVDYE
>seqY535 = Y family seq 535  0.932740 36 Y
YMYRLGKYYYYFHYDYLRYFYYSSYYYYAIYDYYYYYTQYYYYDTVYIETYYHHYIYQYTYGGYYGELMRYEHYELN
>seqC545 = C family seq 545  0.941423 38 C
CKYCACCCGPCICCLTCCCCCCCCRCCCCCPVCTNFCFCCVCDAHCCCCMNPKYYQCAHCQCCCCCLC
>seqK590 = K family seq 590  0.951817 34 K
MIKFKYKVWKGVFKKVPKKKEVVKKKKKKEKYKKVPKSKKDKNMKTNHVWQLSNHQKKSDQDTDTKQVNNDKCKKKKINYKKK
